include/clap/ext/draft/midi-info.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#include "../../plugin.h"
+
+// This extension lets a plugin expose it's MIDI implementation.
+// The primary goal is to allow a host to create a MIDI-CI Property Exchange "ChCtrlList" resource for the plugin,
+// so keyboard controllers can assign their knobs to suitable controllers.
+
+//TODO : clap_midiinfo could contain more (optional) info (min/max, stepcount, paramPath etc.)
+//TODO : for completeness this should work per-input port. Is that worth it?

```

Comment:
yes. it is easier to add port now than have to retrofit it later, and all our other APIs have port.